docs/memcoin.md

```diff
@@ -0,0 +1,60 @@
+# Memcoin
+
+Memcoin is the default CLI to handle Dela nodes. You can find it in
+"cli/node/memcoin". Be sure to run "go install" in it.
+
+```sh
+# Run three nodes
+LLVL=info memcoin --config /tmp/node1 start --port 2001
+LLVL=info memcoin --config /tmp/node2 start --port 2002
+LLVL=info memcoin --config /tmp/node3 start --port 2003
+
+# Share the certificate
+memcoin --config /tmp/node2 minogrpc join \
+    --address 127.0.0.1:2001 $(memcoin --config /tmp/node1 minogrpc token)
+memcoin --config /tmp/node3 minogrpc join \
+    --address 127.0.0.1:2001 $(memcoin --config /tmp/node1 minogrpc token)
+
+# Create a new chain with the three nodes
+memcoin --config /tmp/node1 ordering setup\
+    --member $(memcoin --config /tmp/node1 ordering export)\
+    --member $(memcoin --config /tmp/node2 ordering export)\
+    --member $(memcoin --config /tmp/node3 ordering export)
+
+# Create a bls signer to sign transactions. Be sure you have the "crypto" binary
+# by running "go install" in cli/crypto.
+crypto bls signer new --save private.key
+crypto bls signer read --path private.key --format BASE64
+
+# Authorize the signer to handle the access contract on each node
+memcoin --config /tmp/node1 access add \
+    --identity $(crypto bls signer read --path private.key --format BASE64_PUBKEY)
+memcoin --config /tmp/node2 access add \
+    --identity $(crypto bls signer read --path private.key --format BASE64_PUBKEY)
+memcoin --config /tmp/node3 access add \
+    --identity $(crypto bls signer read --path private.key --format BASE64_PUBKEY)
+
+# Update the access contract to allow us to use the value contract
+memcoin --config /tmp/node1 pool add\
+    --key private.key\
```

Comment:
This should be the absolute path to `private.key`. I got the following error when I tried the relative path as in the example:

```
command error: failed to get signer: failed to load signer: while opening file: open private.key: no such file or directory
```

Reply:
Thank you for trying it out. It is possible to use an absolute, and the path is relative to where the node has been started. I added a note about it.